### ISSUE TITLE
Tiktok doesn't get blocked when timer runs out

### ISSUE THREAD

**Issue by fassina2** [ofekmiz/Habitica-Pomodoro-SiteKeeper#72]
Site blocking doesn't seem to work on tiktok, it says the site is added but it doesn't block it.

**Comment by ofekmiz** [comment]
Hi @fassina2 , it works for me, which browser are you using? Mind sharing a screen recording of the issue? Thanks

**Comment by fassina2** [comment]
I'm on vivaldi. Here's a picture:
![1627939876492](https://user-images.githubusercontent.com/36776815/127926657-79ec53a5-f6f2-43cc-89b1-fe9388efbdec.jpg)

As you can see the site is blocked, I'm not on break and it's not being 'blocked'.

**Comment by fassina2** [comment]
Ha! Ironically I fixed it, apparently if you block a site without adding a cost to unlock it it doesn't block it ?

I noticed tiktok had a 0 cost there, made it 100 and it started blocking the page. 

That's cool, happy it's fixed. BTW can you make starting breaks after typing the duration by pressing enter possible? It's annoying having to click once to start typing, then type, then click on the play button.

**Comment by ofekmiz** [comment]
@fassina2 Great! having a cost 0 will block the website only when the timer is running (on work session), having a cost greater than zero will block it all the time and you will have to pay with habitica gold to unblock it.
The default hot key to activate the timer is **ALT + P**, you can change it in tour browser settings (not sure how to do it in vivaldy but in chrome it's here: chrome://extensions/shortcuts)

**Comment by ofekmiz** [comment]
By the way are you using the timer for work sessions or only for custom breaks by using this:
![image](https://user-images.githubusercontent.com/39601942/127968924-c5f30436-eb38-42df-b107-db0b3a73660b.png)

**Comment by fassina2** [comment]
Yes custom break timer, that's what I use the most. It's quite useful =D
